managed/services/checks/checks.go

```diff
@@ -1153,20 +1210,12 @@ func convertVMValue(value model.Value) ([]byte, error) {
 	return res, nil
 }
 
-func fillQueryPlaceholders(query string, target services.Target) (string, error) {
+func fillQueryPlaceholders(query string, data interface{}) (string, error) {
```

Comment:
Instead interface let's make this function accept some structure that contains all possible fields that we use.
Something like
```
type QueryParameters struct {
                ServiceName string
                ServiceID string
		NodeName    string
}
```

Reply:
In one place we will fill only `ServiceName` and `ServiceID`, in another `ServiceName` and `NodeName`.